build/Dockerfile

```diff
@@ -26,6 +26,18 @@ RUN git clone --single-branch --branch release-4.6 https://github.com/openshift/
 WORKDIR ovn-kubernetes/go-controller/
 RUN make windows
 
+# Build promu utility tool, needed to build windows_exporter.exe metrics binary
+WORKDIR /build/
+RUN git clone --single-branch --branch v0.6.1 https://github.com/prometheus/promu
+WORKDIR /build/promu/
+RUN make build
+
+# Build windows_exporter
+WORKDIR /build/
+RUN git clone --single-branch --branch v0.13.0 https://github.com/prometheus-community/windows_exporter
+WORKDIR /build/windows_exporter/
+RUN GOOS=windows /build/promu/promu build -v
```

Comment:
Given we are going the sub-module route with #144, please move to using that here also.